examples/sample-demos/openebs-zfs-deploy/sc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: storage.k8s.io/v1
```

Comment:
Each example should have a documentation, I could not find

Reply:
yes, Doc PR will be raised referencing these files